backend/src/resolvers/users.spec.js

```diff
@@ -35,34 +36,39 @@ describe('userMiddleware', () => {
     })
   })
 
-  describe('update User', () => {
+  describe('UpdateUser', () => {
+    beforeEach(async () => {
+      await factory.create('User', { id: 'u47', name: 'John Doe' })
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-10T15:42:22Z" title="Friday, May 10th 2019, 5:42:22 pm +02:00">May 10, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -35,34 +36,39 @@ describe('userMiddleware', () => {
     })
   })
 
-  describe('update User', () => {
+  describe('UpdateUser', () => {
+    beforeEach(async () => {
+      await factory.create('User', { id: 'u47', name: 'John Doe' })
```


@ulfgebhardt you tried to update a user that did not exist.